dev-python/slimit/slimit-0.8.1.ebuild

```diff
@@ -0,0 +1,32 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python2_7 pypy )
+inherit distutils-r1
+
+DESCRIPTION="A JavaScript minifier written in Python"
+HOMEPAGE="https://slimit.readthedocs.io/en/latest/"
+SRC_URI="https://github.com/rspivak/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="test"
+
+RDEPEND="dev-python/ply[${PYTHON_USEDEP}]"
+DEPEND="${RDEPEND}
+	dev-python/setuptools[${PYTHON_USEDEP}]
```

Comment:
It uses `entry_points`, so setuptools is RDEP as well.

Reply:
I missed it, thanks for spotting this. Will upload soon, as well.